app/src/main/java/ai/elimu/vitabu/ui/storybook/ChapterFragment.java

```diff
@@ -194,6 +196,12 @@ public void onClick(@NonNull View widget) {
                                     // Report learning event to the Analytics application (https://github.com/elimu-ai/analytics)
                                     LearningEventUtil.reportWordLearningEvent(word, LearningEventType.WORD_PRESSED, getContext(), BuildConfig.ANALYTICS_APPLICATION_ID);
                                 }
+
+                                @Override
+                                public void updateDrawState(@NonNull TextPaint ds) {
```

Comment:
Rename `ds` to something more descriptive? I wasn't able to understand this abbreviation.